modules/nf-core/truvari/bench/tests/main.nf.test

```diff
@@ -0,0 +1,141 @@
+nextflow_process {
+
+    name "Test Process TRUVARI_BENCH"
+    
+    script "../main.nf"
+    process "TRUVARI_BENCH"
+   
+    tag "modules"
+    tag "modules_nfcore"
+    tag "truvari"
+    tag "truvari/bench"
+   
+    test("without bed") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [id:'test'],
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/vcf/chr21/simulated_sv.vcf.gz', checkIfExists:true),
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/vcf/chr21/simulated_sv.vcf.gz.tbi', checkIfExists:true),
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/vcf/chr21/simulated_sv2.vcf.gz', checkIfExists:true),
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/vcf/chr21/simulated_sv2.vcf.gz.tbi', checkIfExists:true),
+                    []
+                    ]
+                input[1] = [
+                    [id:'fasta'],
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/genome.fasta', checkIfExists:true)
+                    ]
+                input[2] = [
```

Comment:
I'm not particularly fond of having the index in a separate channel of the fasta.
But it's not a deal breaker.